src/deploy.sh

```diff
@@ -363,6 +358,8 @@ function kernel_uninstall()
       # We need to update grub, for this reason we to load specific scripts.
       . "$KW_PLUGINS_DIR/kernel_install/$distro.sh" --source-only
       . "$KW_PLUGINS_DIR/kernel_install/utils.sh" --source-only
+      # XXX: Rename kernel_uninstall in the plugin, this name is super
```

Comment:
I think instead of XXX it would be better to just write TODO, so we can easilly find all TODOS in the code (or open an issue in github and keep the code clean)